plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -109,11 +113,16 @@ public function __construct() {
 				'is_experimental' => true,
 				'disable_ui'      => false,
 			),
-			'custom_order_tables'  => array(
-				'name'            => __( 'High-Performance order storage (COT)', 'woocommerce' ),
-				'description'     => __( 'Enable the high performance order storage feature.', 'woocommerce' ),
-				'is_experimental' => true,
-				'disable_ui'      => false,
+			// Options for HPOS features are added in CustomOrdersTableController to keep the logic in same place.
+			'custom_order_tables'  => array( // This exists for back-compat, otherwise is always enabled and hidden from WC 8.0.
+				'name'               => '',
```

Comment:
Shall we give it a name?

```suggestion
				'name'               => __( 'High-Performance order storage (COT)', 'woocommerce' ),
```

Could also add back the description unless it is no longer used, but the name is definitely needed in some parts of the UI:

<div align="center"><img width="800" alt="Screenshot 2023-07-10 at 11 41 15" src="https://github.com/woocommerce/woocommerce/assets/3594411/728e3952-d6b0-4e20-a64c-eb69380039b9"></div>